### ISSUE TITLE
sockaddr/test_uct_sockaddr.err_handle/4 failed on hpc-test-node-gpu02

### ISSUE THREAD

**Issue by dmitrygx** [openucx/ucx#4346]
```
13:04:42 [ RUN      ] sockaddr/test_uct_sockaddr.err_handle/4
13:04:42 [     INFO ] Testing fcfc:fcfc:209:44:225:90ff:fe94:755c:32895 Interface: eth0
13:04:42 [     INFO ] < send(fd=35) failed: Connection reset by peer >
13:04:42 [     INFO ] < failed to send client info: Input/output error >
13:04:42 /scrap/jenkins/workspace/hpc-ucx-pr-6/label/hpc-test-node-gpu/worker/3/contrib/../test/gtest/common/test.cc:273: Failure
13:04:42 Failed
13:04:42 Got 1 warnings during the test
13:04:42 
[  FAILED  ] sockaddr/test_uct_sockaddr.err_handle/4, where GetParam() = sockaddr/eth0 (1 ms)
```

http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/12875/label=hpc-test-node-gpu,worker=3/console

**Comment by yosefe** [comment]
hopefully `sysctl net.core.somaxconn=2048 net.core.netdev_max_backlog=10000 net.ipv4.tcp_max_syn_backlog=10000` should resolve this

**Comment by dmitrygx** [comment]
> hopefully `sysctl net.core.somaxconn=2048 net.core.netdev_max_backlog=10000 net.ipv4.tcp_max_syn_backlog=10000` should resolve this

yes, I agree.
maybe add retries for connection establishment in SOCKSM as well as we have in TCP? wdyt?

**Comment by yosefe** [comment]
> > hopefully `sysctl net.core.somaxconn=2048 net.core.netdev_max_backlog=10000 net.ipv4.tcp_max_syn_backlog=10000` should resolve this
> 
> yes, I agree.
> maybe add retries for connection establishment in SOCKSM as well as we have in TCP? wdyt?

Good idea, but let's do this in next revamp of sockcm, when we move it to use new connection establishment API in UCT

**Comment by dmitrygx** [comment]
> Good idea, but let's do this in next revamp of sockcm, when we move it to use new connection establishment API in UCT

Sure. Since we will fix it by increasing system limits, should I close this ticket then?

**Comment by yosefe** [comment]
> > Good idea, but let's do this in next revamp of sockcm, when we move it to use new connection establishment API in UCT
> 
> Sure. Since we will fix it by increasing system limits, should I close this ticket then?

Let's close it if we don't see this issue again for a while

**Comment by dmitrygx** [comment]
> Let's close it if we don't see this issue again for a while

👍
